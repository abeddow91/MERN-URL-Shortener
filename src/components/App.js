import React, {Component} from 'react';

import Header from './Header';


class App extends Component {
  render () {
    return (
      <div className="App">
      <Header message = "URL Shortener" />
      <div>
      </div>
      </div>

    );
  }
}


export default App;
