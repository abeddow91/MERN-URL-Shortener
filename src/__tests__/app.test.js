import React from 'react';
import ReactDOM from 'react-dom';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '../components/App';

describe('render', () =>{
  it('should render the app', () => {
    const app = shallow(<App />);
    const tree = toJson(app);
    expect(tree).toMatchSnapshot();
  });
});
