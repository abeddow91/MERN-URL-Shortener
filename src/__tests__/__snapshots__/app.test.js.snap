// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render the app 1`] = `
<div
  className="App"
>
  <Header
    message="Url Shortener"
  />
  <div>
    <urlForm />
  </div>
</div>
`;
